src/classwork/FormatDisplay.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function FormatDisplay({ formattedText, formattedTextRef }) {
+  return <h2 ref={formattedTextRef}><pre>{formattedText}</pre></h2>;
```

Comment:
```suggestion
  return <h2 ref={formattedTextRef}>{formattedText}</h2>;
```
This is preventing your header font from being updated